src/core/tsi/alts_transport_security.cc

```diff
@@ -24,19 +24,15 @@
 
 static alts_shared_resource g_alts_resource;
 
-alts_shared_resource* alts_get_shared_resource(void) {
-  return &g_alts_resource;
-}
-
 static void grpc_tsi_alts_wait_for_cq_drain() {
```

Comment:
If this now requires holding `g_alts_resource.mu`, how about adding a `_locked` suffix to the name?